literature.bib

```diff
@@ -83,6 +98,57 @@ @inproceedings{STS:VaMoS20
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2019 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{PAP+:ICST19,
+  title={Uniform sampling of sat solutions for configurable systems: Are we there yet?},
+  author={Plazar, Quentin and Acher, Mathieu and Perrouin, Gilles and Devroey, Xavier and Cordy, Maxime},
+  booktitle={2019 12th IEEE Conference on Software Testing, Validation and Verification (ICST)},
+  pages={240--251},
+  month={April},
+  year={2019},
+  doi={10.1109/ICST.2019.00032},
+  sampling-tags = {name=Uniform sampling, testing, non-functional properties, feature model, available tool, open-source tool, implementation artifacts, sampling efficiency, effectiveness, uses QuickSampler/UniGen},
```

Comment:
non-functional properties really used as input to the sampling algorithm?

uniform sampling has been done by multiple papers and we need to have unique names for each algorithm

Reply:
Have a look at the other names. I have come up with names too and you could use a similar scheme.

Ideally, you would order tags same as on the website to avoid such confusion.

Reply:
Check